libs/scripts/scripts.js

```diff
@@ -36,9 +36,11 @@ const stageDomainsMap = {
   },
   '.business-graybox.adobe.com': { 'business.adobe.com': 'origin' },
   // TODO: remove after QE approval
-  '--milo--robert-bogos.hlx.page': {
-    'www.adobe.com': 'main--dc--adobecom.hlx.page',
-    'business.adobe.com': 'business.stage.adobe.com',
+  '^https://.*--milo--robert-bogos.hlx.page': {
```

Comment:
Does this need to be added to the unit tests? Or is it picked up from the scripts

Reply:
Is not picked up from scripts. I added unit test for it 👍